### ISSUE TITLE
Some code cells rendering as horizontal lines, not showing code

### ISSUE THREAD

**Issue by pllim** [jupyterlab/jupyterlab#11078]
## Description

![Screenshot 2021-09-14 152645](https://user-images.githubusercontent.com/2090236/133321402-5137aa0d-ceaf-4dad-9b22-9a525d5474f5.jpg)

```json
    "imviz.load_static_regions(my_regions)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "82be4828",
   "metadata": {},
   "source": [
    "You can also programmatically control the viewer."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1ad5d8d1",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Center the image on given pixel position.\n",
    "viewer.center_on((1173, 1013))  # X, Y (0-indexed)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1a5920da",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Move the image with the given pixel offsets.\n",
    "viewer.offset_by(500, -100)  # dX, dY"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "09f17716",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Center the image on given sky coordinates.\n",
    "sky = SkyCoord('00h24m07.33s -71d52m50.71s')\n",
    "viewer.center_on(sky)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "703dfb79",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Move the image with the given sky offsets.\n",
    "viewer.offset_by(0.5 * u.arcsec, -1.5 * u.arcsec)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f0cda8bd",
   "metadata": {},
   "source": [
    "You can programmatically zoom in and out.\n",
    "\n",
    "Zoom level:\n",
    "\n",
    "* 1 means real-pixel-size.\n",
    "* 2 means zoomed in by a factor of 2.\n",
    "* 0.5 means zoomed out by a factor of 2.\n",
    "* 'fit' means zoomed to fit the whole image width into display."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "459e16a0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get the current zoom level.\n",
    "viewer.zoom_level"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f3694c2b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set the zoom level directly.\n",
    "viewer.zoom_level = 1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5bc0fa9a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set the relative zoom based on current zoom level.\n",
    "viewer.zoom(2)"
   ]
  },
```

## Reproduce

I am not sure how to consistently trigger this. It comes and goes... 🤯 When when it happens, it is consistently the same cells.

I tried wiping "id" fields from the cells and let Lab regenerate them, the problem went away for a bit, and then came back anyway.

## Expected behavior

Code cells to render normally.

## Context

- Operating System and version: Windows 10 + WSL2 (Debian)
- Browser and version: Chrome Version 93.0.4577.63 (Official Build) (64-bit)
- JupyterLab version: 3.1.9

**Comment by jasongrout** [comment]
There were some issues like this fixed in the 3.1.10 or 3.1.11 release. Can you upgrade and let us know if you still see the problem?

**Comment by pllim** [comment]
Thanks for the quick reply! I updated to `jupyterlab` 3.1.12 and the problem seems to be gone for now. If it comes back, I'll open a new issue. 😄
